livekit-agents/livekit/agents/stt/stt.py

```diff
@@ -76,9 +88,12 @@ async def recognize(
         event = await self._recognize_impl(buffer, language=language)
         duration = time.perf_counter() - start_time
         stt_metrics: STTMetrics = {
+            "request_id": event.request_id,
             "timestamp": time.time(),
             "duration": duration,
             "label": self._label,
+            "audio_duration": calculate_audio_duration(buffer),
```

Comment:
any differences between this and what's contained in `ReconitionUsage`?

Reply:
Yeah the only difference is that RecognitionUsage is used by the subclasses, the metrics emitted doesn't use this class